src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -220,7 +220,8 @@ private void CreatePullReplicationAsHub(TcpConnectionOptions tcpConnectionOption
 
             var taskId = pullReplicationDefinition.TaskId; // every connection to this pull replication on the hub will have the same task id.
             var externalReplication = pullReplicationDefinition.ToExternalReplication(initialRequest, taskId);
-            var outgoingReplication = new OutgoingReplicationHandler(this, Database, externalReplication, external: true, initialRequest.Info)
+
+            var outgoingReplication = new OutgoingReplicationHandler(tcpConnectionOptions, this, Database, externalReplication, external: true, initialRequest.Info)
```

Comment:
IIRC Those `tcpConnectionOptions` are already added in `DispatchDatabaseTcpConnection`.
This is pull replication, meaning the _destination_ opens the tcp connection to the source.